js/app.js

```diff
@@ -1 +1,261 @@
+        announceWinner.innerText = "Player1 is the winner!";
+        //alert("Game is over!");
+    } else if (seven === player1 && eight === player1 && nine === player1){
+        announceWinner.innerText = "Player1 is the winner!"; 
+        //alert("Game is over!");
+    } else if (one === player1 && four === player1 & seven === player1){
+        announceWinner.innerText = "Player1 is the winner!"; 
+        //alert("Game is over!");
+    } else if (two === player1 && five === player1 & eight === player1){
+        announceWinner.innerText = "Player1 is the winner!"; 
+        //alert("Game is over!");
+    } else if (three === player1 && six === player1 & nine === player1){
+        announceWinner.innerText = "Player1 is the winner!"; 
+        //alert("Game is over!");
+    } else if (one === player1 && five === player1 & nine === player1){
+        announceWinner.innerText = "Player1 is the winner!"; 
+        //alert("Game is over!");
+    } else if(three === player1 && five === player1 & seven === player1){
+        announceWinner.innerText = "Player1 is the winner!"; 
+        //alert("Game is over!");
+    }
+
+    if (one === player2 && two === player2 && three === player2) {
+        announceWinner.innerText = "Player2 is the winner!";
+    } else if (four === player2 && five === player2 && six === player2){
+        console.log("Player2 is the winner!");
+        announceWinner.innerText = "Player2 is the winner!";
+    } else if (seven === player2 && eight === player2 && nine === player2){
+        console.log("Player2 is the winner!");
+        announceWinner.innerText = "Player2 is the winner!"; 
+    } else if (one === player2 && four === player2 & seven === player2){
+        console.log("Player2 is the winner!");
+        announceWinner.innerText = "Player2 is the winner!";
+    } else if (two === player2 && five === player2 & eight === player2){
+        console.log("Player2 is the winner!");
+        announceWinner.innerText = "Player2 is the winner!";
+    } else if (three === player2 && six === player2 & nine === player2){
+        console.log("Player2 is the winner!");
+        announceWinner.innerText = "Player2 is the winner!";
+    } else if (one === player2 && five === player2 & nine === player2){
+        console.log("Player2 is the winner!");
+        announceWinner.innerText = "Player2 is the winner!";
+    } else if (three === player2 && five === player2 & seven === player2){
+        console.log("Player2 is the winner!"); 
+        announceWinner.innerText = "Player2 is the winner!";
+    }
+
+
+    function alertWhenClickFilledBox() {
+        let square = document.querySelector(".square");
+        if (square.textContent === player1 || square.textContent === player2) {
+            alert("This square is already filled. please choose another one");
+        }
+    }
+    document.querySelector(".square").addEventListener("click", alertWhenClickFilledBox);
```

Comment:
Line 233 adds an event listener to the first square after the first move is played (because this code is inside of your checkWin1 function).  You likely meant to add an event listener to each square after it was selected but this code is not doing that.  There are other ways to handle dealing with preventing a square from being played a second time. 
 You'll notice that you're still able to play on the square because the event listeners that you created on lines 22-156 are still present.  Two possible solutions for solving this problem are: 1) You could change the code so that you could remove those event listeners from 22-156 or 2) You could create logic inside of those event listeners that prevents a square that has been played from updating (and also avoid updating the current turn in that case as well).